tests/integration_cases/test_raft.rs

```diff
@@ -2979,11 +2985,14 @@ fn test_remove_node() {
     setup_for_test();
     let mut r = new_test_raft(1, vec![1, 2], 10, 1, new_storage());
     r.remove_node(2);
-    assert_eq!(r.prs().nodes(), vec![1]);
+    assert_eq!(
+        r.prs().voter_ids().iter().cloned().collect::<Vec<_>>(),
```

Comment:
Better keep consistent style with above.

Reply:
It' the wrong data type. I'll just take the `next()` and unwrap the value.